packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -336,18 +325,7 @@ class OverlayState extends State<Overlay> with TickerProviderStateMixin {
   ///
   /// It is an error to specify both `above` and `below`.
   void insertAll(Iterable<OverlayEntry> entries, { OverlayEntry below, OverlayEntry above }) {
-    assert(
-      above == null || below == null,
-      'Only one of `above` and `below` may be specified.',
-    );
-    assert(
-      above == null || (above._overlay == this && _entries.contains(above)),
-      'The provided entry for `above` is not present in the Overlay.',
-    );
-    assert(
-      below == null || (below._overlay == this && _entries.contains(below)),
-      'The provided entry for `below` is not present in the Overlay.',
-    );
+    _assertEntriesInsertedPosition(above, below);
```

Comment:
can you pass in _entries here?

Reply:
and I will wrap this into another assert so that we don't spend time call into another method in production code.